src/services/storage/handlers/offer.ts

```diff
@@ -66,6 +66,8 @@ const handler: Handler<StorageServices> = {
 
     await offer.save()
 
+    // @TODO if we receive an unknown message then update actually doesn't happens, because offer don't changed
```

Comment:
Good catch, although I think it is not such a big deal if the `updated` event is emitted. Especially since this should not happen often. I guess leaving the TODO there, for now, is sufficient.

Reply:
Ok